config/locales/en.yml

```diff
@@ -86,3 +86,21 @@ en:
 
   project_module_gtt: "GTT"
   permission_manage_gtt_settings: "Manage GTT settings"
+
+  title_geojson_upload: "Import GeoJSON"
+  placeholder_geojson_upload: "Please paste a GeoJSON geometry here, or import the GeoJSON data from a file."
```

Comment:
@dkastl These items are in modal dialog, so it may be good to move the bottom `gtt_js/modal/`.